### PR TITLE
Add hybrid Neon/Neon Keccak-f1600, don't use scalar ASM on Cortex-A72

### PR DESCRIPTION
**Pull request by hanno-becker**

### CODE REVIEW

File: fips202/asm/aarch64/keccak_f1600_x2_v8a_v84a_hyrid.S
Comment (mkannwischer) [inline fips202/asm/aarch64/keccak_f1600_x2_v8a_v84a_hyrid.S:1]: There is a typo in the file name here

Comment (hanno-becker) [reply fips202/asm/aarch64/keccak_f1600_x2_v8a_v84a_hyrid.S:1]: Fixed

**Comment by hanno-becker** [comment]
Benchmarks suggest that something isn't yet working as intended. Both Gv2 and Gv3 get slower.

**Review by mkannwischer — APPROVED** [review]
Thanks! This looks great and works on my machine. 

We somehow need to add to the CI output which implementations is actually used, but we can do that separately. I'll open an issue for that.

**Pull request merged**